Vagrantfile

```diff
@@ -17,9 +17,6 @@ Vagrant.configure('2') do |config|
 
   config.vm.provision :chef_solo do |chef|
     chef.json = {
-      :riemann => {
-        :version => '0.2.4'
```

Comment:
Is this used anywhere to override the default?  It looks like you took it out of the chef recipe, so are still using the 0.2.10 package?